src/openms/include/OpenMS/ANALYSIS/MAPMATCHING/ConsensusMapNormalizerAlgorithmMedian.h

```diff
@@ -61,16 +61,19 @@ namespace OpenMS
     virtual ~ConsensusMapNormalizerAlgorithmMedian();
 
     /**
- * @brief normalizes the maps of the consensusMap
+     * @brief normalizes the maps of the consensusMap
      * @param map ConsensusMap
+     * @param shift whether to shift to same median instead of scaling
      */
-    static void normalizeMaps(ConsensusMap & map);
+    static void normalizeMaps(ConsensusMap & map, bool shift);
```

Comment:
just a personal taste: one could use an 
enum NormalizationMethod { MN_scale, MN_shift }, to make the shift parameter more descriptive.